app/components/views/Bond/index.tsx

```diff
@@ -749,10 +754,39 @@ export const Bond: FC<Props> = (props) => {
                 <Spinner />
               </div>
             ) : (
-              <ButtonPrimary
-                {...getButtonProps()}
-                className={styles.submitButton}
-              />
+              <Box
+                sx={{ display: "flex", flexDirection: "column", width: "100%" }}
+              >
+                <ButtonPrimary
+                  {...getButtonProps()}
+                  className={styles.submitButton}
+                />
+                {view === "redeem" && !showSpinner && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      width: "100%",
+                      alignItems: "center",
+                      justifyContent: "center",
+                      paddingRight: "16px",
+                    }}
+                  >
+                    <Checkbox
+                      checked={shouldAutostake}
+                      onChange={handleAutostakeCheck}
+                      sx={{ color: "#fff" }}
+                    />{" "}
+                    <Text t="caption" align="center">
+                      <Trans
+                        id="bond.should_autostake"
+                        comment="should autostake?"
+                      >
+                        Autostake
```

Comment:
trying to think if something more translate-able.... Let's go with:
```suggestion
  Automatically stake for sKLIMA
```

Reply:
sounds good